### DISCUSSION TITLE
Where do you keep the clippings?

### DISCUSSION THREAD

**Discussion by aur3l14no** [webclipper/web-clipper#1006] [Q&A] [answered]
~

**Comment by stewstryker** [comment]
I'm also curious if the data is processed locally, then sent directly to the desired destination (e.g. Bear, OneNote, Notion, etc.). I'm a little paranoid about my privacy.

**Comment by DiamondYuan** [comment, marked as answer]
we never store your clippings, sent directly to the desired destination .
